cds-snc/platform-mvp
# Send email to wpforms subscribers

# Summary | Résumé

Adds a send_internal method to send notifications to WPForms subscribers.

Also did a bit of refactoring - adding a namespace to the Notify classes, moving them to `/notify` and setting up the autoloader.

Now, any file that includes the Autoloader (ie, `require __DIR__ . '/vendor/autoload.php';`) will have access to the classes in the `CDS\Notify` namespace.

## To test
See comments on diff for where these changes are needed:

- Update dropdown values to include valid form_id's from your installation
- Call `self::send_internal` from `process_send`
- Make sure you have some "confirmed" subscribers in your wpforms_entries table
